h/admin.py

```diff
@@ -214,6 +214,27 @@ def badge_remove(request):
 def groups_index(request):
     return {"groups": models.Group.all()}
 
+@view.view_config(route_name='admin_groups',
```

Comment:
I reckon we should make the URL for this `/admin/groups.csv`. That would mean giving it its own route name (`admin_groups_csv`) and setting the URL in `includeme`.